src/5e-SRD-Features.json

```
@@ -519,96 +519,96 @@
     "choice": {
       "choose": 2,
       "type": "feature",
-      "from": [
+      "from":  [
```

Dodgy spacing here, same below. Otherwise, LGTM!

```suggestion
      "from": [
```

---

fixed :)